Makefile.am

```diff
@@ -115,15 +116,12 @@ SUBDIRS = .
 DIST_SUBDIRS = . perl/annotator perl/imap perl/sieve/managesieve
 dist_sysconf_DATA =
 lib_LTLIBRARIES = lib/libcyrus_min.la lib/libcyrus.la
-EXTRA_PROGRAMS =
```

Comment:
Generally I'm keeping all these initialisations here at the top (even if they're blank) so that conditional paths later can always-append without needing to chase the original declaration around.

Reply:
EXTRA_PROGAMS is not utilized anyhow in Makefile.am , so there is no need for this variable.